beater/api/intake/handler.go

```diff
@@ -99,6 +99,12 @@ func Handler(handler StreamHandler, requestMetadataFunc RequestMetadataFunc, bat
 	}
 }
 
+func OtlpHandler(handler http.HandlerFunc) request.Handler {
```

Comment:
This function does not belong here. "api/intake" is just for the Elastic APM  agent intake protocol.

Reply:
Removed in 51cdb119d